tools/release.sh

```diff
@@ -13,7 +13,23 @@ webuser=dist
 promotablecmd=dist-promotable
 promotecmd=dist-promote
 signcmd=dist-sign
-
+customsshkey="" # let ssh and scp use default key
+signversion=""
+
+while getopts "i:s:" option; do
+    case "${option}" in
+        i)
+            customsshkey="-i ${OPTARG}"
+            ;;
+        s)
+            signversion="${OPTARG}"
+            ;;
+        \?)
+            exit 1
```

Comment:
If you want a nicer error message you can do this:

```diff
diff --git a/tools/release.sh b/tools/release.sh
index 4225be4cc7..fcb5ef1242 100755
--- a/tools/release.sh
+++ b/tools/release.sh
@@ -16,7 +16,7 @@ signcmd=dist-sign
 customsshkey="" # let ssh and scp use default key
 signversion=""
 
-while getopts "i:s:" option; do
+while getopts ":i:s:" option; do
     case "${option}" in
         i)
             customsshkey="-i ${OPTARG}"
@@ -25,6 +25,11 @@ while getopts "i:s:" option; do
             signversion="${OPTARG}"
             ;;
         \?)
+            echo "Invalid option -$OPTARG."
+            exit 1
+            ;;
+        :)
+            echo "Option -$OPTARG takes a parameter."
             exit 1
             ;;
     esac

```

up to you whether it's worth it though.